tests/unit/package_managers/oci/test_oci_client.py

```
@@ -0,0 +1,154 @@
+            select_platform(index, platform, "test/repo")
+
+    def test_error_lists_available_platforms(self) -> None:
+        index = self._make_index(
+            [
+                {"os": "linux", "architecture": "amd64"},
+                {"os": "linux", "architecture": "arm64"},
+            ]
+        )
+        platform = OciPlatform(os="windows", architecture="amd64")
+
+        with pytest.raises(FetchError, match="linux/amd64.*linux/arm64"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)